project/apps/user/src/app/app.module.ts

```diff
@@ -1,9 +1,21 @@
 import { Module } from '@nestjs/common';
+import { ConfigModule } from '@nestjs/config';
 import { AuthenticationModule } from '@project/authentication';
 import { BlogUserModule } from '@project/blog-user';
+import { UserConfigModule } from '@project/config';
+import { PrismaClientModule } from '@project/models';
 
 @Module({
-  imports: [BlogUserModule, AuthenticationModule],
+  imports: [
+    ConfigModule.forRoot({
+      envFilePath: ['libs/user/models/.env', 'apps/user/user.env', '.env'],
```

Comment:
Попробуй так, может быть сработает:
```suggestion
      envFilePath: ['./.env'],
```